doc/library/compile/mode.txt

```diff
@@ -23,7 +23,7 @@ Theano defines the following modes by name:
 - ``'DebugMode'``: A mode for debugging. See :ref:`DebugMode <debugmode>` for details.
 - ``'ProfileMode'``: Deprecated, use the Theano flag :attr:`config.profile`.
 - ``'DEBUG_MODE'``: Deprecated. Use the string DebugMode.
-- ``'PROFILE_MODE'``: Deprecated. Use the string ProfileMode.
+- ``'PROFILE_MODE'``: Deprecated. Use the flag profile.
```

Comment:
Put a reference to the profile flag (just like 2 lines above, simply copy that whole description and paste it here)